bin/README.md

```diff
@@ -0,0 +1,29 @@
+# funclist
+
+usage:
+```
+./funclist api.json
+```
+
+Look in a OpenApi json file, the list of Functions.
+To do so, look at which object in components.schemas end up with Request.
+Then each functions call are separate by a space
+
+# line_check
+
+usage:
+```
+./line_check "test_keywork" "this_arg" <<< "there is this_arg in this line"
+```
+
+Will return "this_arg".
+
+This utilise was made so we don't have to call grep for each keywork we need to look at.
+With this utility we can just give all our keyword to line_check, and then switch on which keywork was found.
+
+This is also a lot faster that multiples calls to grep
+
+# osc-api-seems-valid.sh
+
+This utilise is use to check is osc-api.json have been correctly generate.
+It look if the file is indeed a json, and if it contain at last one API function.
```

Comment:
This utility is used to check if osc-api.json has been generated correctly. It verifies that the file is a valid JSON and contains at least one API function.